source/administration/hipchat-migration-guidelines.rst

```diff
@@ -39,9 +39,11 @@ Step 3: Import your data into Mattermost
 
 1. Follow the `Mattermost Bulk Load Tool <https://docs.mattermost.com/deployment/bulk-loading.html>`__ guide to import your data into Mattermost. Files exported from HipChat will need to be converted to the format required by Mattermost. Please `contact us <https://mattermost.zendesk.com/hc/en-us/requests/new>`__ if you require assistance in the conversion. 
 
-  - Note: Efforts are underway to source scripts from the Mattermost community to further automate this step. One solution already submitted by the community is `HipMost <https://github.com/orbitalimpact/hipmost>`__.  If you’re interested in contributing or testing any community contributed solution, please let us know at info@mattermost.com, Twitter or Mattermost forums at https://forum.mattermost.org.
+2. You are also encouraged to use a new HipChat import tool created by Herzum: https://github.com/herzum/HC2MM.
```

Comment:
I'd recommend revising this remove "new" to "You are also encouraged to use the HipChat import tool..." so that the term "new" does not become outdated or discourage people from using something that was recently released.